src/components/calltree/CallTree.js

```diff
@@ -40,10 +40,12 @@ import type {
   CallNodeDisplayData,
   WeightType,
 } from 'firefox-profiler/types';
-import type { CallTree as CallTreeType } from '../../profile-logic/call-tree';
+  
+import type { CallTree } from 'firefox-profiler/profile-logic/call-tree';
```

Comment:
This was `import type { CallTree as CallTreeType } from '../../profile-logic/call-tree';` you should also keep the `as CallTreeType` here. I think this code changed after fixing the merge conflicts. The tests are failing because of it. Also you can remove the empty line above this line.